daemon/containerd/image_commit.go

```diff
@@ -113,12 +111,12 @@ func (i *ImageService) CommitImage(ctx context.Context, cc backend.CommitConfig)
 
 // generateCommitImageConfig returns commit oci image config based on the container's image.
 func generateCommitImageConfig(ctx context.Context, container *containerapi.Config, baseConfig ocispec.Image, diffID digest.Digest, opts backend.CommitConfig) (ocispec.Image, error) {
-	if opts.Config.Cmd != nil {
-		baseConfig.Config.Cmd = opts.Config.Cmd
-	}
-	if opts.Config.Entrypoint != nil {
-		baseConfig.Config.Entrypoint = opts.Config.Entrypoint
-	}
+	// if opts.Config.Cmd != nil {
+	// 	baseConfig.Config.Cmd = opts.Config.Cmd
+	// }
+	// if opts.Config.Entrypoint != nil {
+	// 	baseConfig.Config.Entrypoint = opts.Config.Entrypoint
+	// }
```

Comment:
Was this commented-out for debugging, still "TODO", or "to be removed"?

Reply:
Mostly commented out so I would remember to call it out for reviews and look at it a bit more to make sure that this change isn't breaking anything 😅 This came about because @rumpl realized that we were generating incorrect configs (specifically with `CMD` and `ENTRYPOINT`) due to the way commit was previously inheriting those from the base image vs using the container config from the `CommitConfig`.

Reply:
- Opened a PR with this fix; https://github.com/moby/moby/pull/45438 (added you both as "co-authored-by"